Stripe/STPAddCardViewController.m

```diff
@@ -644,4 +676,13 @@ - (UIView *)tableView:(__unused UITableView *)tableView viewForFooterInSection:(
     }
 }
 
+- (void)useShippingAddress:(UIButton *)sender {
+    self.addressViewModel.address = self.shippingAddress;
+    self.hasUsedShippingAddress = YES;
+    [[self firstEmptyField] becomeFirstResponder];
+    [UIView animateWithDuration:0.2f animations:^{
+        sender.alpha = 0;
+    }];
```

Comment:
Since this code changes the sizeThatFits / heightForHeaderInSection value, you need to do `[tableView beginUpdates]; [tableView endUpdates]` so that the table view will immediately recalc the height and not get into a bad state later.

Also I think changing the alpha of the sender here and then relying on that value for calculations in the actual class is a little iffy. It would be better to have the hiding logic inside the header view class itself and a property to change whether it is showing the button or not.